src/Microsoft.DotNet.Build.Tasks/PackageFiles/buildagainstpackages.targets

```diff
@@ -177,6 +177,11 @@
     <PropertyGroup>
         <PackageBuildNumberOverride Condition="'$(PackageBuildNumberOverride)' == '' and '$(_DerivedPackageNumber)' != ''">$(_DerivedPackageNumber)</PackageBuildNumberOverride>
     </PropertyGroup>
+    <!-- we don't yet have runtime packages for these RIDs so exclude them for now -->
+    <ItemGroup>
+      <ExcludedRuntimes Include="fedora.23-x64" />
```

Comment:
So this version: `1.0.1-rc4-24126-06` makes me think it's expecting packages from somewhere else (maybe what was just built from the build pipeline?

We definitely publish packages for builds we do out of TFS (see https://dotnet.myget.org/feed/dotnet-core/package/nuget/runtime.ubuntu.16.04-x64.runtime.native.System.Net.Security)